config.pri

```diff
@@ -56,7 +56,7 @@ CONFIG(no_opengl) {
 }
 
 # default value for comic archive decompression backend
-unix:!macx:!CONFIG(unarr):!CONFIG(7zip) {
+unix:!macx:!CONFIG(unarr):!CONFIG(7zip):!CONFIG(libarchive) {
   CONFIG += unarr
 }
 
```

Comment:
The win32 and macx branches need to check if libarchive is set in the config too. If this is not done, config.pri will add the respective defaults to the config and libarchive will be silently ignored or - worse - we have a conflict because more than one decompression backend is specified.